test/get_has_memory.tcl

```diff
@@ -0,0 +1,11 @@
+# Tests whether the has_memory attribute works for cells and libcells
+read_liberty ../examples/gf180mcu_sram.lib.gz
```

Comment:
this lib is not in the examples dir

Reply:
Sorry, I'm confused. https://github.com/parallaxsw/OpenSTA/pull/129/files#diff-71319827d46ce51b81049eef16dce8bb178cc42ebdfd540e0bc8673ad24379c1

Do you want me to move it to the test directory?

Reply:
weird, it doesn't show up in the patch file. but I think it would be better in the test dir because it isn't used by an example

Reply:
Sure I can move it rn